config/vendor-functions/vx-certifier.sh

```diff
@@ -1,100 +1,96 @@
 #!/usr/bin/env bash
 
-# A script to create a machine cert from a cert signing request (CSR) on a USB drive, as written by
+# A script to create machine certs from cert signing requests (CSRs) on a USB drive, as written by
 # create-machine-cert.sh.
 #
-# Usage: sudo VX_PRIVATE_KEY_PATH=/path/to/vx-private-key.pem /path/to/vx-certifier.sh
+# Usage:
+#
+# # Read from a USB drive written to by a machine
+# sudo VX_PRIVATE_KEY_PATH=/path/to/vx-private-key.pem /path/to/vx-certifier.sh
+#
+# # Read from the specified directory
+# sudo VX_PRIVATE_KEY_PATH=/path/to/vx-private-key.pem CERTS_DIRECTORY=/path/to/certs-directory /path/to/vx-certifier.sh
```

Comment:
As noted in the PR description:

> Now that we handle the overwhelming majority of VxCertifier requests via a remote file exchange, it's a bit cumbersome for Adam and I to have to download the file onto a USB drive first. This PR introduces an env var that can be used to direct the VxCertifier script to read from a specified directory rather than a USB drive.